frontend/package.json

```diff
@@ -4,8 +4,11 @@
   "private": true,
   "proxy": "http://127.0.0.1:8000",
   "dependencies": {
+    "bootstrap": "^5.2.3",
```

Comment:
when the first engineering decisions where made for OSmOSE applications, we chose to restrict the number of libraries used in order to keep the website size as light as possible. As the number of people working on those applications grows, I understand that we will have to allow some libraries (like Bootstrap here) to ease the developer experience. But please keep in mind that adding a library always come with a cost (size, maintenance, security issues...)